nbs/15_gsa_gls/20-null_simulations/05_gwas/07_03-pca-plot_select.ipynb

```diff
@@ -0,0 +1,1063 @@
+{
```

Comment:
<p>You are only selecting one of the EUR clusters right? In the quadrant at the lower left.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Ah, yes, that's actually the main cluster, a big one at the lower left corner, the other one is a just a small group of samples that I'm excluding just in case.</p>